translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml

```diff
@@ -1,205 +1,256 @@
-            ecosystems: __components.crosslinks.types.ecosystems
-            event: __components.crosslinks.types.event
-            events: __components.crosslinks.types.events
-            news: __components.crosslinks.types.news
-            pressrelease: __components.crosslinks.types.pressrelease
-            pressreleases: __components.crosslinks.types.pressreleases
-            page: __components.crosslinks.types.page
+            blogpost: 博客
+            ecosystem: 生态系统
+            ecosystems: 生态系统
+            event: 事件
+            events: 事件
+            news: 新闻
+            pressrelease: 媒体新闻稿
+            pressreleases: 媒体新闻稿
+            page: 页面
 post:
     meta:
         credit: >
-            __post.meta.credit
-        credit_title: __post.meta.credit_title
+            由:
+        credit_title: 作者及出版日期
         published: >
-            __post.meta.published
-        skiplink: __post.meta.skiplink
-    notes_heading: __post.notes_heading
+            出版:
+        skiplink: 跳至第{count}条评论
+    notes_heading: 注释说明
     related:
-        heading: __post.related.heading
-        heading_tags: __post.related.heading_tags
-        heading_ecosystems: __post.related.heading_ecosystems
-        heading_groups: __post.related.heading_groups
-        heading_specifications: __post.related.heading_specifications
+        heading: 相关内容
+        heading_tags: 标签
+        heading_ecosystems: 生态系统
+        heading_groups: 组
+        heading_specifications: 技术标准
     feed:
-        title: __post.feed.title
-        link: __post.feed.link {feed_type}
+        title: 相关的 RSS
+        link: 订阅我们的{feed_type}信息源
     comments:
-        count: __post.comments.count <span>{count}</span>
-        skip_to_form: __post.comments.skip_to_form
-        reply: __post.comments.reply <span class="visuallyhidden">{name}</span>
-        reply_attr: __post.comments.reply_attr {name}
+        count: 第<span>({count})</span>条评论
+        skip_to_form: 跳跃至评论区
+        reply: 回复<span class="visuallyhidden">{name}</span>
+        reply_attr: 回复{name}
         form:
-            title: __post.comments.form.title
-        closed: __post.comments.closed
+            title: 发布新评论
+        closed: 该贴的评论区已关闭。
 notes:
     info:
-        default_title: __notes.info.default_title
+        default_title: 信息
     successes:
-        default_title: __notes.successes.default_title
+        default_title: 成功
     warnings:
-        default_title: __notes.warnings.default_title
+        default_title: 警示
     errors:
-        default_title: __notes.errors.default_title
+        default_title: 出现了一些问题
 events:
     # same_day = 1 when start_date == end_date
     date_range: >-
         {same_day, select,
             1  {{start_date}, {start_time} - {end_time} <b>{timezone}</b>}
-            other {{start_date}, {start_time} - {end_date}, {end_time} <b>{timezone}</b>}
+            其它 {{start_date}, {start_time} - {end_date}, {end_time} <b>{timezone}</b>}
```

Comment:
"other" cannot be translated, it's part of a condition:
 - if same_day is 1, use the first translation
 - if same_day is something else (other), use the second translation